### PR TITLE
archisteamfarm: Add version 4.0.4.3

### PR DESCRIPTION
**Pull request by davidxuang**
Adding [ArchiSteamFarm](https://github.com/davidxuang/scoop-extras/tree/patch/archisteamfarm). Rename `.zip` to `.7z` as PowerShell does not support Deflate64.

### CODE REVIEW

File: bucket/archisteamfarm.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "##": "Rename from zip to 7z is needed due to 'unsupported compression method' error.",
+    "version": "4.0.4.3",
+    "description": "Farm Steam cards from multiple accounts simultaneously.",
+    "homepage": "https://github.com/JustArchiNET/ArchiSteamFarm",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/JustArchiNET/ArchiSteamFarm/releases/download/4.0.4.3/ASF-win-x64.zip#force.7z",
```
Comment (Ash258) [inline bucket/archisteamfarm.json]:
```suggestion
            "url": "https://github.com/JustArchiNET/ArchiSteamFarm/releases/download/4.0.4.3/ASF-win-x64.zip#/dl.7z",
```

File: bucket/archisteamfarm.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        [
+            "ArchiSteamFarm.exe",
+            "ArchiSteamFarm"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/JustArchiNET/ArchiSteamFarm/releases/download/$version/ASF-win-x64.zip#force.7z"
```
Comment (Ash258) [inline bucket/archisteamfarm.json]:
```suggestion
                "url": "https://github.com/JustArchiNET/ArchiSteamFarm/releases/download/$version/ASF-win-x64.zip#/dl.7z"
```

**Comment by davidxuang** [comment]
@Ash258 Should `bin` be kept along with `shortcuts`?

**Comment by Ash258** [comment]
Manifest is OK.
I cannot approve it as I think software, which is using exploits and other gray level practices should not be inside official buckets.

**Comment by davidxuang** [comment]
Fair enough.

**Comment by Ash258** [comment]
I do not see a reason to close this PR. This is my personal opinion, which is not shared between real collaborators.

**Comment by davidxuang** [comment]
Not so sure with it... Maybe I'd just leave it here.

**Pull request closed without merging**